flight/flight-sql-jdbc-core/src/main/java/org/apache/arrow/driver/jdbc/accessor/impl/calendar/ArrowFlightJdbcTimeStampVectorAccessor.java

```diff
@@ -58,6 +71,7 @@ public ArrowFlightJdbcTimeStampVectorAccessor(
     this.holder = new Holder();
     this.getter = createGetter(vector);
 
+    this.isZoned = getVectorIsZoned(vector);
     this.timeZone = getTimeZoneForVector(vector);
```

Comment:
It's a bit strange that both the timeZone and isZoned field exist and would be good to clarify why/how they are used and are different.

Reply:
Yeah, originally I thought about setting tz to null if not available but there's some expectations about `timeZone` not being null and having some default value in a few places where it felt like untangling would make this too big. I'll add some comments.